codebuild/bin/install_default_dependencies.sh

```diff
@@ -100,16 +117,28 @@ if [[ "$SAW" == "true" || "$TESTS" == "ALL" ]]; then
     codebuild/bin/install_z3_yices.sh "$(mktemp -d)" "$Z3_INSTALL_DIR" > /dev/null ;
 fi
 
-if [[ "$TESTS" == "benchmark" || "$TESTS" == "ALL" ]]; then
-    # Install tox if running on Ubuntu(only supported Linux at this time)
-    if [[ "$OS_NAME" == "linux" && ! -x `which cmake` ]]; then
+if [[ ! -x `which cmake` ]]; then
+    case "$DISTRO" in
+    "ubuntu")
         apt-get -y install cmake
-    fi
+        ;;
+    "redhat"|"fedora"|"centos"|"amazon linux")
```

Comment:
Do we every treat these OSes differently? Is it worth distinguishing between them rn?

Reply:
They each can have their own package naming schemes and available versions. It might be too early, but also time to start thinking about it...

Reply:
A long list of platforms we don't necessarily have regular builds for seems like an opportunity for mistakes. If we forget one in any of these switch statements, we won't find out until someone tries to build on that platform. It might be safer to keep it simple until you need more complexity.